app/code/Magento/Catalog/view/frontend/web/js/validate-product.js

```diff
@@ -41,6 +41,7 @@ define([
                     return false;
                 }
             });
+            $('.action.tocart').attr('disabled',false);
```

Comment:
Such selector is not acceptable, see https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-demarcation.html#business-logic-and-javascript

Reply:
@orlangur 

fixed it, please check it.